app/Resources/translations/messages.it.xliff

```diff
@@ -0,0 +1,241 @@
+<?xml version="1.0"?>
+<xliff version="1.2" xmlns="urn:oasis:names:tc:xliff:document:1.2">
+    <file source-language="en" target-language="it" datatype="plaintext" original="file.ext">
+        <body>
+            <trans-unit id="note">
+                <source>note</source>
+                <target>NOTE</target>
+            </trans-unit>
+            <trans-unit id="tip">
+                <source>tip</source>
+                <target>SUGGERIMENTO</target>
+            </trans-unit>
+            <trans-unit id="not_available">
+                <source>not_available</source>
+                <target>Non disponibile</target>
+            </trans-unit>
+            <trans-unit id="mit_license">
+                <source>mit_license</source>
+                <target>Licenza MIT</target>
+            </trans-unit>
+
+            <trans-unit id="title.homepage">
+                <source>title.homepage</source>
+                <target><![CDATA[Benvenuto nell'applicazione <strong>Symfony Demo</strong>]]></target>
+            </trans-unit>
+            <trans-unit id="title.source_code">
+                <source>title.source_code</source>
+                <target>Codice sorgente usato per effettuare il render di questa pagina</target>
+            </trans-unit>
+            <trans-unit id="title.controller_code">
+                <source>title.controller_code</source>
+                <target>Codice del controller</target>
```

Comment:
Personally I don't agree much with `controllore`, because to me in Italian it has the sense of strictly "performing controls" or "overseeing", whereas a controller in a Symfony app does more than that, and the foreign term helps conveying that different meaning.
However, in order to keep it consistent with the docs, I can just change that.